server/.ebextensions/00-environment.config

```diff
@@ -1,5 +1,3 @@
 option_settings:
-  aws:elasticbeanstalk:container:nodejs:
-    GzipCompression: true
-    NodeCommand: "npm run start:prod"
+  aws:elasticbeanstalk:environment:proxy:
```

Comment:
Are we sure this will not break old envs?

Reply:
and production as well, I believe...
Seems too much of a risk, I don't think we will have migrated staging and prod in the next sprint 🤔 

There might be a way to make it backward compatible, don't you think?

Reply:
Maybe a different script can be run for migrated envs and non-migrated ones